core/api/src/main/java/com/alipay/sofa/rpc/ext/ExtensionLoader.java

```diff
@@ -212,6 +212,10 @@ protected void readLine(URL url, String line) {
                 LOGGER.warn("Extension {} of extensible {} is disabled, cause by: {}",
                     className, interfaceName, ExceptionUtils.toShortString(e, 2));
             }
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("Extension {} of extensible {} is disabled, cause by: {}",
+                    className, interfaceName, ExceptionUtils.toString(e));
```

Comment:
LOGGER.debug("Extension " + className + " of extensible " + interfaceName + " is disabled.", e);